modules/client/src/controllers/WithdrawalController.ts

```diff
@@ -67,7 +66,8 @@ export class WithdrawalController extends AbstractController {
       }
       const postWithdrawBalances = await this.connext.getFreeBalance(assetId);
 
-      const expectedFreeBal = preWithdrawBalances[myFreeBalanceAddress].sub(amount);
+      console.log(`***** preWithdrawBalances: ${stringify(preWithdrawBalances)}`);
```

Comment:
We should probably prefer `this.log` over `console`

Reply:
oh we should definitely take that out that's not supposed to be in there 🙃